source/sass/ckeditor/_ckeditor5.scss

```diff
@@ -1,6 +1,6 @@
 // Import this file in a ckeditor5.scss file in your sub theme.
 // See starterkit in this theme as example.
-@use "sass:meta";
+@use "sass:meta" as *;
```

Comment:
Better to load a module as is and use its namespace. It makes it more clear where the mixin is comming from.

BTW: we should check if we can't create a module out of the styleguide.
It would make the @use statements for the styleguide parts easier...